bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -1479,11 +1479,15 @@ void DWARFRewriter::patchLowHigh(DWARFDie DIE, DebugAddressRange Range,
     TempDebugPatcher->addLE64Patch(LowPCOffset, Range.LowPC);
   }
 
+  uint64_t HighPC = Range.HighPC;
+  if (HighPCVal->V.getForm() == dwarf::Form::DW_FORM_data8 ||
```

Comment:
I think high_pc can also be data2 and data1.
Would it be better to check for DW_FORM_addr since we know that is absolute and treat rest as relative?

Reply:
Yes, I was thinking about this too, but forgot to do this, thank you!